torii/build/dsl.py

```diff
@@ -272,6 +272,6 @@ def __repr__(self) -> str:
 	def __len__(self) -> int:
 		return len(self.mapping)
 
-	def __iter__(self) -> Generator[tuple[str, str], None, None]:
+	def __iter__(self) -> Generator[tuple[str, str]]:
```

Comment:
Think this introduces a bug - recall that we had to add the `, None, None` to get correct typing on older Pythons (3.10, 3.11) and that these parameters are not optional on them.

Reply:
This passes in 3.10 and 3.11, CI for those versions came back clean, and I tested in a local py3.10 version and it was happy enough.

Reply:
Re-trying it locally, despite its prior protestations, now agrees. Okie.